### PR TITLE
Expose CheckResponse as wolfSSL_CertManagerCheckOCSP_Staple

### PR DESCRIPTION
**Pull request by kojo1**
for Ticket #4834
Changed name CertManagerCheckOCSP_Staple to CertManagerCheckOCSPResponse

### CODE REVIEW

File: src/ocsp.c [hunk 1]
```diff
@@ -383,6 +383,24 @@ static int CheckResponse(WOLFSSL_OCSP* ocsp, byte* response, int responseSz,
     return ret;
 }
 
+WOLFSSL_API int wolfSSL_CertManagerCheckOCSP_Staple(WOLFSSL_CERT_MANAGER *cm,
+                                                    byte *response, int responseSz, buffer *responseBuffer,
```
Comment (ejohnstown) [inline src/ocsp.c]: Whitespace change request: could you shift these lines to the left so they don't go past column 80? Changing these lines to an indent of 8 spaces would be good.

File: wolfssl/ocsp.h [hunk 2]
```diff
@@ -54,7 +54,9 @@ WOLFSSL_LOCAL int  CheckCertOCSP_ex(WOLFSSL_OCSP*, DecodedCert*,
                              WOLFSSL_BUFFER_INFO* responseBuffer, WOLFSSL* ssl);
 WOLFSSL_LOCAL int  CheckOcspRequest(WOLFSSL_OCSP* ocsp,
                  OcspRequest* ocspRequest, WOLFSSL_BUFFER_INFO* responseBuffer);
-
+WOLFSSL_API int wolfSSL_CertManagerCheckOCSP_Staple(WOLFSSL_CERT_MANAGER *,
```
Comment (ejohnstown) [inline wolfssl/ocsp.h]: Whitespace change request: could you shift these lines to the left so they don't go past column 80? Changing these lines to an indent of 8 spaces would be good.

File: src/ocsp.c (hunk 1 above)
Comment (kojo1) [reply src/ocsp.c]: Fixed

File: wolfssl/ocsp.h (hunk 2 above)
Comment (kojo1) [reply wolfssl/ocsp.h]: Fixed

**Comment by kojo1** [comment]
Changed name CertManagerCheckOCSP_Staple to CertManagerCheckOCSPResponse.
Moved them from ocsp.[ch] to ssl.[ch].

**Pull request merged**